packages/eslint-config-react-app/base.js

```diff
@@ -29,8 +29,18 @@ module.exports = {
   parserOptions: {
     ecmaVersion: 2018,
     sourceType: 'module',
-    ecmaFeatures: {
-      jsx: true,
+    requireConfigFile: false,
+    babelOptions: {
+      parserOpts: {
+        plugins: [
+          ['flow', { all: true, enums: true }],
```

Comment:
What is the extra config now needed for?

Reply:
`babel-eslint` 10 enable these parser options by default: 

https://github.com/babel/babel-eslint/blob/4bd049e06e2c7ab31053020fc662f51bf6c179e3/lib/parse.js#L21-L51

But `@babel/eslint-parser` requires a babel config, _or_ in this case since the Babel config is hidden in react-scripts, we provide our own parser options.

Reply:
Thanks for the context. This config here is quite a bit more limited than what we had previously it looks like. The lack of things like `nullishCoalescingOperator`, `optionalChaining`, `decorators-legacy`, etc. make me a bit nervous.

Our babel config is here: https://github.com/facebook/create-react-app/tree/master/packages/babel-preset-react-app

Reply:
I have reset to parser plugins because

1) `babel-preset-react-app` accepts different options (e.g. Flow/TypeScript), these options impacts the required parser syntax plugins
2) In `babel-preset-react-app`, `decorators-legacy` is applied for `tsx?` files, which is handled by `typescript-eslint` instead of `@babel/eslint-parser`. In other words, if CRA users used legacy decorators in `.jsx?`, error will be thrown from `babel-loader`.
3) Based on CRA use case I don't think the other parser plugins in `babel-eslint` are required: They are stage 1&2 proposals which is _not_ supported by `babel-preset-react-app`. If CRA users use these syntaxes, parsing error will be thrown from `babel-loader`.

Q: Can `eslint-config-react-app` be used independently from CRA? If so, removing parser plugins can be a breaking change for these users and I am happy to bring back these plugins.

Reply:
Yes the eslint config is most definitely used elsewhere: https://www.npmjs.com/package/eslint-config-react-app

Reply:
Added the other ES proposals enabled by `babel-eslint@10` we should consider drop them in future versions.